app/src/main/res/values/strings.xml

```diff
@@ -49,8 +49,10 @@
     <string name="quest_openingHours_add_hours">"Add hours"</string>
     <string name="quest_openingHours_add_weekdays">"Add weekdays"</string>
     <string name="quest_openingHours_chooseMonthsTitle">"Select months"</string>
-    <string name="quest_openingHours_chooseWeekdaysTitle">"Select weekdays"</string>
+    <string name="quest_openingHours_chooseWeekdaysTitle">"Select group of weekdays with the same opening hours"</string>
```

Comment:
Does this button still fit on the screen on small (4.3") screens?

Reply:
If it does not fit or potentially does not fit in other languages, and the text does not wrap but is cut off, you should be able to change the style of the title bar by modifying `DialogWindowTitle.Bubble` in `styles.xml` (`android:maxLines`)

Reply:
Not sure. I tried one handed mode to simulate this, but it is scaling screen.

It is anyway in WIP for related reason as translations of just 120% length are getting cutoff on 5.0" screen.

Reply:
You can use an emulator or just set a longer test and see if it wraps. If it wraps, it is okay.

Reply:
It gets cutoff with "...".

Reply:
> you should be able to change the style of the title bar by modifying DialogWindowTitle.Bubble in styles.xml (android:maxLines)

Unfortunately, with

```
    <style name="DialogWindowTitle.Bubble" parent="android:Widget">
        <item name="android:textAppearance">@style/TextAppearance.Title</item>
        <item name="android:paddingBottom">8dp</item>
        <item name="android:maxLines">3</item>
    </style>
```

it is cutoff at 5.0" screen as follows:

`Select group of weekdays with the same opening hoursSelect group of ...`

I thought about switching from PopupMenu to full screen menu but first I want to try https://stackoverflow.com/a/48161007/4130619

Reply:
What does the PopupMenu have to do with this? Isn't this about the title text of the weekday picker dialog?